runtime/caml/major_gc.h

```diff
@@ -62,7 +62,6 @@ void caml_orphan_allocated_words();
 void caml_add_to_orphaned_ephe_list(struct caml_ephe_info* ephe_info);
```

Comment:
Because it is no longer used elsewhere, can we remove `caml_ephe_todo_list_emptied` from the interface exposed by `major_gc.h`?